app/pages/lab/workflow.cjsx

```diff
@@ -15,7 +15,7 @@ WorkflowCreateForm = require './workflow-create-form'
 workflowActions = require './actions/workflow'
 classnames = require 'classnames'
 
-`import ShortcutEditor from '../../classifier/tasks/shortcut-editor';`
+`import ShortcutEditor from '../../classifier/tasks/shortcut/shortcut-editor';`
```

Comment:
Should the editor be renamed to `classifier/tasks/shortcut/editor` for clarity?

Reply:
Yes please